src/main/java/pl/piekoszek/gorskimatches/facebook/FacebookController.java

```diff
@@ -1,27 +1,36 @@
 package pl.piekoszek.gorskimatches.facebook;
 
-import org.springframework.http.ResponseEntity;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.http.HttpStatus;
 import org.springframework.web.bind.annotation.*;
 
 @RestController
-@RequestMapping("web")
+@RequestMapping("api/webhook/facebook/page/message")
 public class FacebookController {
 
     private FacebookMessageService facebookMessageService;
 
-    FacebookController(FacebookMessageService facebookMessageService){
+    @Value("${VERIFY_TOKEN}")
```

Comment:
Set some default value or make it optional so app can start without `VERIFY_TOKEN` env variable set